src/component/Tooltip.tsx

```diff
@@ -90,99 +99,100 @@ export type TooltipProps<TValue extends ValueType, TName extends NameType> = Def
   useTranslate3d?: boolean;
 };
 
-export class Tooltip<TValue extends ValueType, TName extends NameType> extends PureComponent<
-  TooltipProps<TValue, TName>
-> {
-  static displayName = 'Tooltip';
-
-  static defaultProps = {
-    active: false,
-    allowEscapeViewBox: { x: false, y: false },
-    reverseDirection: { x: false, y: false },
-    offset: 10,
-    viewBox: { x1: 0, x2: 0, y1: 0, y2: 0 },
-    coordinate: { x: 0, y: 0 },
-    cursorStyle: {},
-    separator: ' : ',
-    wrapperStyle: {},
-    contentStyle: {},
-    itemStyle: {},
-    labelStyle: {},
-    cursor: true,
-    trigger: 'hover',
-    isAnimationActive: !Global.isSsr,
-    animationEasing: 'ease',
-    animationDuration: 400,
-    filterNull: true,
-    useTranslate3d: false,
-  };
-
-  state = {
-    boxWidth: -1,
-    boxHeight: -1,
-    dismissed: false,
-    dismissedAtCoordinate: { x: 0, y: 0 },
-  };
-
-  private wrapperNode: HTMLDivElement;
-
-  componentDidMount() {
-    this.updateBBox();
-  }
-
-  componentWillUnmount() {
-    document.removeEventListener('keydown', this.handleKeyDown);
-  }
-
-  componentDidUpdate() {
-    this.updateBBox();
-  }
-
-  handleKeyDown = (event: KeyboardEvent) => {
-    if (event.key === 'Escape') {
-      this.setState({
-        dismissed: true,
-        dismissedAtCoordinate: {
-          ...this.state.dismissedAtCoordinate,
-          x: this.props.coordinate.x,
-          y: this.props.coordinate.y,
-        },
-      });
-    }
-  };
+const tooltipDefaultProps: TooltipProps<number, string> = {
```

Comment:
typed as number, string for default props. Doesn't really affect anything except for typing this explicitly